app/controllers/browse_controller.rb

```diff
@@ -44,6 +44,8 @@ def subject_list_params
     end
 
     def title_list_params
+      params[:prefix] = params[:prefix].gsub(/^(?i)(an|a|the) /, '')
```

Comment:
I think this needs to be `&.gsub`. When I go to `All` in the browse, it throws a nil error.

Reply:
Ah right, I'll add that.